### DISCUSSION TITLE
Named views at root

### DISCUSSION THREAD

**Discussion by Ericlm** [posva/unplugin-vue-router#485] [Questions and Help] [answered]
I have the following `pages/index.vue`:
```vue
<template>
  <RouterView name="sidebar" />

  <RouterView />
<template>
```

The pages are displayed correctly inside of the `RouterView`, but when creating the file `index@sidebar.vue`, it doesn't get displayed inside of `<RouterView name="sidebar" />`. 
Am I doing something wrong? This would be handy when using named views at `index.vue` 🙂.

**Comment by posva** [comment]
This is the correct way, it works when I tested.

**Reply by Ericlm** [reply]
Could you tell me what component you put as the "App" for Vue ? I set index.vue like so: `import App from './pages/index.vue'` but I'm not sure about this.

**Reply by Ericlm** [reply]
Alright, I understood what was the issue. In fact, I wanted to have nested routes (a folder in which I know the user is authenticated, so I can put a sidebar eveywhere), but without nesting routes (so we can access "/dashboard", "/settings" directly, instead of having something like "/home/dashboard"). 

Don't know if there is a way to nest layouts without nesting routes ? The opposite of "Nested routes without nesting layouts" 😅

**Reply by posva** [reply, marked as answer]
use `definePage()` within the page component to change the route path

**Reply by Ericlm** [reply]
Thank you! As there is a lot of pages I made it in the `extendRoute` in the config ; thank you for your help! (and the amazing plugin :)
